CHANGES.rst

```diff
@@ -1,6 +1,23 @@
 Changes
 =======
 
+Version 0.6.0 (2019-12-20)
+--------------------------
+
+- Adds new flag and configuration to optionally deploy with CERN storage EOS.
+- Creates a new Kubernetes service account for REANA and associates the
+  persmissions REANA needs.
```

Comment:
`Creates a new Kubernetes service account for REANA and associates the permissions to the REANA needs.`
or 
`Creates a new Kubernetes service account for REANA with the proper permissions`